.swagger-revision

```diff
@@ -1 +1 @@
-62fbcb9e8bec4a40569e51c7f316af85806628ee
+v2.2.0
```

Comment:
Using tags is not yet a supported feature of our `script/swagger`, it will re-download, clean, and rebuild the libs on every execution. This adds about 30 seconds to each execution.